test/e2e/scheduling/preemption.go

```diff
@@ -24,14 +24,15 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	appsv1 "k8s.io/api/apps/v1"
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Let's revert this change since import `core/v1` as `v1` is a convention in k/k, and I don't see the value of updating the alias in this PR which could bring many noises.

Reply:
Ok, just wasn't sure why it was even aliased to `v1` in the first place